data/remnant/remnant jobs.txt

```diff
@@ -307,14 +309,15 @@ mission "Remnant: Logistics Outfitter Delivery"
 		dialog phrase "generic cargo on visit"
 	on complete
 		dialog `A repair technician inspects the <cargo> parts and authorizes your payment of <payment>.`
-		payment 30000 25
+		payment
+		payment 10000
```

Comment:
This one goes from about 35000 to 40000. (calculated for 100 cargo)

Reply:
This one especially seems like it should have `payment 15000` or so in order to make it better in some way than the previous one. This might just be me, but I feel like it should be more valuable somehow.